homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/ball/Magic.java

```diff
@@ -0,0 +1,60 @@
+package com.tutrit.java.quickstart.homework.ball;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class Magic {
+    private final Random random = new Random();
+    private final Map<Integer, String> ball = new HashMap<>();
```

Comment:
I think what you need rename it variable
ball->predicts

Reply:
Thanks. I think this name is good. I will not change it.